pepper-apis/docs/specification/src/components/schemas.yml

```diff
@@ -114,49 +113,59 @@ Activity:
   required:
     - activityType
     - formType
+    - activityCode
     - guid
+    - title
```

Comment:
Are these values guaranteed to be present in the response, or are we considering them nullable?

Reply:
`title` is guaranteed and non-null.

Reply:
Looking at [DDP-4378](https://broadinstitute.atlassian.net/browse/DDP-4378), both the `title` and `subtitle` are mention to be `optional`, not required, which doesn't seem to mesh with this. Are these the properties being referred to in the ticket?

Reply:
They're optional as in study does not have to define it. If `title` is not defined, it will be blank. `subtitle` is nullable.

Reply:
I believe this is addressed. Resolving comment.